### PR TITLE
Major changes in the quantms package

### PR DESCRIPTION
**Pull request by ypriverol**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
  - [x] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by daichengxin — APPROVED** [review]

File: modules/local/diannconvert/main.nf [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
+        --fasta "${fasta}" \\
+        --charge $params.max_precursor_charge \\
+        --missed_cleavages $params.allowed_missed_cleavages \\
+        --qvalue_threshold $params.protein_level_fdr_cutoff \\
+        |& tee convert_report.log
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        sdrf-pipelines: \$(echo "0.0.21")
+        pyopenms: \$(echo "2.8.0")
```
Comment (jpfeuffer) [inline modules/local/diannconvert/main.nf:46]: We should print the actual pyopenms version from the command line.

**Review by jpfeuffer — APPROVED** [review]
Small comment. Can be done later also.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### `Changed`
+
+- [#169](https://github.com/bigbio/quantms/pull/169) - Restruct DIA-NN step1 : Generate an in silico predicted spectral library
+- [#178](https://github.com/bigbio/quantms/pull/178) - Restruct DIA-NN step2 : Preliminary analysis of individual raw files
+- [#179](https://github.com/bigbio/quantms/pull/179) - Restruct DIA-NN steps 3-5 to be as parallel as possible
+
 ### `Fixed`
 
 ### `Dependencies`
```
[the remark this answers is not in the document]
Comment (jpfeuffer) [reply CHANGELOG.md:20]: Under dependencies. Also openms.

**Comment by ypriverol** [comment]
@jpfeuffer can I merge this?

**Comment by jpfeuffer** [comment]
Is the pyopenms version fixed? You can also make an issue for this.

**Comment by ypriverol** [comment]
> Is the pyopenms version fixed? You can also make an issue for this.

@daichengxin can you fix this?

**Pull request merged**